config/initializers/simple_form_bootstrap.rb

```diff
@@ -1,142 +1,4 @@
 # Use this setup block to configure all options available in SimpleForm.
-SimpleForm.setup do |config|
-  config.error_notification_class = 'alert alert-danger'
-  config.button_class = 'btn btn-default'
-  config.boolean_label_class = nil
+SimpleForm::Bootstrap.setup do |config|
```

Comment:
Unused block argument - `config`. You can omit the argument if you don't care about it.